server/events/metrics/common.go

```diff
@@ -28,4 +28,13 @@ const (
 
 	// Signal receive is when we receive it off the channel
 	SignalReceive = "signal_receive"
+
+	// Metrics are scoped to workflow namespaces anyways so let's
+	// keep these metrics simple.
+	WorkflowSuccess = "success"
+	WorkflowFailure = "failure"
+	WorkflowLatency = "latency"
```

Comment:
Are there tags to determine if the workflow in question is deploy vs. terraform?

Reply:
It's embedded as a namespace using an inbound interceptor:
https://github.com/lyft/atlantis/blob/release-v0.17.3-lyft.1/server/neptune/temporal/worker.go#L122